### DISCUSSION TITLE
❓ Question: save_audio but for video.

### DISCUSSION THREAD

**Discussion by timurguseynov** [snakers4/silero-vad#510] [Q&A]
Hi guys,

Thank you for this awesome library :)

May I ask you how it can be used to get the same results as save_audio does but for videos?

**Comment by snakers4** [comment]
> May I ask you how it can be used to get the same results as save_audio does but for videos?

Just use `ffmpeg` to extract the audio track.

**Reply by timurguseynov** [reply]
I mean to use it to cut the videos itself instead of just audio :) Kind of an automatic jumpcut like https://github.com/emkademy/jumpcutter or https://github.com/WyattBlue/auto-editor

I was thinking about creating multiple cuts using `speech_timestamps` and concatenating them after, or using `ffmpeg`'s `filter_complex`, but I thought maybe you know a better way to do it?

**Reply by timurguseynov** [reply]
@snakers4
